### PR TITLE
Update multiple Simple execute test for reliablility

### PR DESCRIPTION
**Pull request by anthonydresser**
Instead of assuming all queries will be there to await in the test, I just await on the queries that are there. This removes the index out of bounds possibility.

### CODE REVIEW

File: src/Microsoft.SqlTools.ServiceLayer/QueryExecution/ResultOnlyContext.cs [hunk 1]
```diff
@@ -15,10 +15,10 @@ namespace Microsoft.SqlTools.ServiceLayer.QueryExecution
     /// </summary>
     public class ResultOnlyContext<TResult> : IEventSender
     {
-        private readonly RequestContext<TResult> origContext;
+        private readonly RequestContext<TResult> OrigContext;
```
Comment (llali) [inline src/Microsoft.SqlTools.ServiceLayer/QueryExecution/ResultOnlyContext.cs]: why did you rename this?

Comment (anthonydresser) [inline src/Microsoft.SqlTools.ServiceLayer/QueryExecution/ResultOnlyContext.cs]: Its unrelated, I just noticed it wasn't consistent with the rest of our c# code
